Configure

```diff
@@ -179,6 +179,7 @@ my $apitable = {
 our %table = ();
 our %config = ();
 our %withargs = ();
+our $now_printing;
```

Comment:
Considering its ad-hoc hack nature, perhaps a comment line or two to explain it?

Reply:
Like pushed?